python/mxnet/gluon/parameter.py

```diff
@@ -646,7 +647,7 @@ def get_constant(self, name, value=None):
             assert isinstance(param, Constant), \
                 "Parameter '{}' already exists but it is not a constant.".format(
                     name)
-            if isinstance(value, nd.NDArray):
+            if isinstance(value, mx.nd.NDArray):
```

Comment:
`nd` has been imported as `ndarray` so you could use `ndarray.NDArray` I think to remain consistent with how it's been done in this file